src/app/common/FabricTableImpl.h

```diff
@@ -0,0 +1,143 @@
+/**
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <lib/support/TypeTraits.h>
+
+namespace chip {
+namespace app {
+namespace common {
+
+/**
+ * @brief Implementation of a storage in nonvolatile storage of a templatized table that stores by fabric id.
+ *
+ * DefaultSceneTableImpl is an implementation that allows to store scenes using PersistentStorageDelegate.
```

Comment:
This comment needs to be updated for the Specifics of FabricTableImpl

Reply:
Thanks, missed this one, done